apps/web/src/hooks/useApplicationInfo.ts

```diff
@@ -0,0 +1,31 @@
+import { useSearchParams } from "react-router-dom";
+
+import { Scalars } from "~/api/generated";
+
+import useRoutes from "./useRoutes";
+
+const useApplicationInfo = (userId?: Scalars["ID"]) => {
```

Comment:
You know me... could you add some documentation to this hook?  :laughing:

Reply:
Thanks for reminding me 😄 Added in [add documentation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5683/commits/7ad4ced1697a72c0cb71fabfc49dc965735ef029)